snippets/csharp/VS_Snippets_WebNet/CustomServerControlCollectionProperties/projectsample/customservercontrolcollection/samples.aspnet.cs/quickcontacts.cs

```diff
@@ -36,10 +36,7 @@ public ArrayList Contacts
         {
             get
             {
-                if (contactsList == null)
-                {
-                    contactsList = new ArrayList();
-                }
+				contactsList ??= new ArrayList();
```

Comment:
another tabs vs. spaces:

```suggestion
                contactsList ??= new ArrayList();
```